scene/main/viewport.cpp

```diff
@@ -1618,7 +1618,8 @@ void Viewport::_gui_show_tooltip() {
 	gui.tooltip_popup->set_position(r.position);
 	gui.tooltip_popup->set_size(r.size);
 
-	gui.tooltip_popup->show();
+	if (window->has_focus())
+		gui.tooltip_popup->show();
```

Comment:
Braces are now required for single-line `if` statements:

```suggestion
	if (window->has_focus()) {
		gui.tooltip_popup->show();
	}

```